python/neuroglancer/write_annotations.py

```diff
@@ -89,17 +213,59 @@ def __init__(
         annotation_type: AnnotationType,
         relationships: Sequence[str] = (),
         properties: Sequence[viewer_state.AnnotationPropertySpec] = (),
+        chunk_size: Union[int, Sequence[int]] = 256,
```

Comment:
chunk_size can actually be a float since the coordinates themselves are floats.  However one complication is that in the future if automatic multiscale and chunking is implemented, which is what our Google-internal pipeline does, there is not much reason to ever specify a chunk size explicitly, and then it is not clear exactly how this parameter would fit in.  Perhaps it could be renamed `experimental_chunk_shape`.

Reply:
fixed

Reply:
yeah i started on a branch to implement the downsampling scheme which starts at the top and sub-divides down as the precomputed protocol suggested.  First i'm getting it working and then plan on doing some significant refactoring. See work in progress here (https://github.com/seung-lab/neuroglancer/blob/annotation_writer_downsample/python/neuroglancer/write_annotations.py).

For now, I've renamed this parameter as you suggested with a note about the anticipated future directions.